docs/functions.rst

```diff
@@ -597,8 +597,8 @@ This function deletes the snapshot data backed up by the BackupData function.
 
    `namespace`, Yes, `string`, namespace in which to execute
    `backupArtifactPrefix`, Yes, `string`, path to the backup on the object store
-   `backupIdentifier`, No, `string`, (required if backupTag not provided) unique snapshot id generated during backup
-   `backupTag`, No, `string`, (required if backupIdentifier not provided) unique tag added during the backup
+   `backupID`, No, `string`, (required if backupTag not provided) unique snapshot id generated during backup
```

Comment:
Thanks for finding this. Please add `backupID` to `spelling_wordlist.txt` for the CI to pass 👍🏼

Reply:
The CI is green now.